File: Ejercicios Resueltos/Lenguaje C/Ejercicios Basicos/Ejercicio-22/ejercicio22.c
// Ejercicio 22
// Solicitar al usuario que ingrese la base y altura de un rectángulo, y calcular y mostrar por
// pantalla el área y perímetro del mismo
// area = base * altura
// perimetro = 2 * altura + 2 * base.

#include <stdio.h>

void main(){
    
    float base, altura, area, perimetro;

    printf("Ingrese el tamaño de la base: ");
    scanf("%f",&base);

    printf("Ingrese el tamaño de la altura: ");
    scanf("%f",&altura);

    area = base * altura;
    perimetro = 2 * altura + 2 * base;

    printf("El area es: %.2f\n", area);
    printf("El perimetro es: %.2f\n", perimetro);

}

File: Ejercicios Resueltos/Lenguaje Python/Ejercicios Basicos/Ejercicio-22/ejercicio22.py
# Ejercicio 22
# Solicitar al usuario que ingrese la base y altura de un rectángulo, y calcular y mostrar por
# pantalla el área y perímetro del mismo
# area = base * altura
# perimetro = 2 * altura + 2 * base.

print("EJERCICIO 22\n\n")

base =  float(input("Ingrese la base en cm del rectangulo: "))
altura = float(input("Ingrese la altura del rectangulo: "))

area = base * altura
perimetro = 2 * altura + 2* base

print(f'\nEl périmetro del rectangulo es de: {perimetro} cm')
print(f'\nEl area del rectangulo es de: {area} cm2')

input("\n\nPresione ENTER para continuar...")